internal/node/bazel_require_script.js

```diff
@@ -0,0 +1,38 @@
+// Adapt node programs to run under Bazel
+// Meant to be run in a --require hook
+
+const fs = require('fs');
+const path = require('path');
+const orig = {};
+// TODO: more functions need patched like
+// the async and native versions
+orig['realPathSync'] = fs.realpathSync;
+orig['lstatSync'] = fs.lstatSync;
+
+// To fully resolve a symlink requires recursively
+// following symlinks until the target is a file
+// rather than a symlink, so we must make this look
+// like a file.
+function lstatSync(p) {
```

Comment:
can we call fs.statSync here instead of lstatSync? 
this would allow it to work as expected for links to directories.

Reply:
discussed in-person: this script is currently the minimum to make rollup work and will need expansion in later PRs as we add tests for things like Jest that exercise the APIs more. @soldair will own it generally.